components/gitpod-protocol/src/protocol.ts

```diff
@@ -1254,4 +1254,15 @@ export interface Terms {
 
 export interface WorkspaceClusterPreference {
     region?: string;
-}
+    // lrm == lastRegionMeasurement. Not writing the full name to save
+    // a few 100mb in the database.
+    lrm?: string;
```

Comment:
Wait, for real? 🤯 (How was this calculated? Also, doesn't MySQL do compression? And sould we start using very short / unintelligible attribute names and values in every common shape? 😬 https://acronyms-suck.com/)